packages/docker-engine/daemon-json

```diff
@@ -1,3 +1,8 @@
+[required-extensions]
+container-registry = "v1"
+oci-defaults = { version = "v1", helpers = ["oci_defaults"] }
```

Comment:
This helper changed significantly in bd2619eaae0240ea0029495787c0e356997a07ae. Just wanted to flag that but I don't think it's a problem.